app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -115,6 +108,14 @@ class HomeMenu(
         val experiments = context.components.analytics.experiments
         val settings = context.components.settings
 
+        val syncSignInMenuItem = BrowserMenuImageText(
+            getSyncItemTitle(),
+            R.drawable.ic_synced_tabs,
+            primaryTextColor
+        ) {
+        onItemTapped.invoke(Item.SyncAccount(accountManager.accountState))
```

Comment:
Nit: `onItemTapped` should be indented
```suggestion
            onItemTapped.invoke(Item.SyncAccount(accountManager.accountState))
```